packages/zoe/index.js

```diff
@@ -0,0 +1,3 @@
+import { makeZoe } from './zoe';
+
+export { makeZoe };
```

Comment:
You're not using the imported name, so drop the `import` line and better to use just:
```js
export { makeZoe } from './zoe';
```

Reply:
Ah, nicely done. I'll change this